monkey/monkey_island/cc/ui/src/components/pages/LandingPage.tsx

```diff
@@ -74,7 +74,7 @@ const LandingPageComponent = (props: Props) => {
   }
 
   function setScenario(scenario: string) {
-    IslandHttpClient.put('/api/island/mode', {'mode': scenario})
+    IslandHttpClient.put('/api/island/mode',  scenario)
```

Comment:
Do you need to serialize `scenario` into JSON?

Reply:
`IslandHttpClient` is doing that